core-cube/src/main/java/org/apache/kylin/cube/CubeSegment.java

```diff
@@ -530,11 +537,19 @@ public void setSnapshots(ConcurrentHashMap<String, String> snapshots) {
     }
 
     public String getStatisticsResourcePath() {
-        return getStatisticsResourcePath(this.getCubeInstance().getName(), this.getUuid());
+        return getStatisticsResourcePath(this.getCubeInstance().getName(), this.getUuid(), ".seq");
+    }
+
+    public String getPreciseStatisticsResourcePath() {
+        return getStatisticsResourcePath(this.getCubeInstance().getName(), this.getUuid(), ".json");
```

Comment:
What's the difference between '.seq' directory and '.json' directory? Do we need to delete which one after running job successfully?